bom/pom.xml

```diff
@@ -362,7 +362,7 @@ THE SOFTWARE.
       <dependency><!-- until we get this version through Stapler -->
         <groupId>org.kohsuke.stapler</groupId>
         <artifactId>json-lib</artifactId>
-        <version>2.4-jenkins-2</version>
+        <version>2.4-jenkins-3</version>
```

Comment:
Seems legitimate, but since this is a custom fork we should review the changes between the two versions. Best to do this in a separate PR accompanied by a link to the GitHub compare view showing the differences.

Reply:
We need to be extremely careful updating this library given how JSON signature validation works. I struggled with unexpectedly broken JSON serialization results a lot in update-center2.